Add-ons/Starter-Kit-TEMP/Widgets/index.md

```diff
@@ -186,3 +186,26 @@ You will have some settings to personalize how your form is presented through th
 - Text Color
 - Form (this is where you add the form that you have already created)
 - Container Width
+
+### The Contact Form Widget
+
+![movePage.jpg](images/Contact-Form.png)
+
+The Contact Form Widget will let you add a contact form to your project, however, to make it work you will need to set up a contact email.
+You set up the contact email by going to the settings tab, then you proceed to the general tab, from here you fill out the ***Contact Form Email***.
+
+![movePage.jpg](images/settings-contact-form.png)
```

Comment:
Instead of having this image here - perhaps we could link to the article describing this setting instead? 🤔 What do you think?